pkg/csi/cephfsdaemonset.go

```diff
@@ -23,13 +23,14 @@ import (
 	"github.com/red-hat-storage/ocs-client-operator/pkg/utils"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
-	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/utils/ptr"
 )
 
 const (
-	CephFSDamonSetName = "csi-cephfsplugin"
+	CephFSDaemonSetName = "csi-cephfsplugin"
+
+	cephFSDamonSetContainerName = "csi-cephfsplugin"
```

Comment:
I don't think we'll ever want the name of deployment and main pod be different, can use same var, same for remaining.

Reply:
The reason I added the container names is because it was not the same on all of them (I believe the culprit is `RDBDeployment`). With that there is no established pattern, so when there is no pattern I fall back to explicit naming across the board.